cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/velocity/listener/VelocityPlayerListener.java

```diff
@@ -72,8 +72,14 @@ public void handle(ServerConnectedEvent event) {
 
     @Subscribe
     public void handle(KickedFromServerEvent event) {
-        VelocityCloudNetHelper.getNextFallback(event.getPlayer())
+        if(!event.getPlayer().isActive())
+            return;
+
+        BridgeProxyHelper.handleConnectionFailed(event.getPlayer().getUniqueId(), event.getServer().getServerInfo().getName());
+
+        VelocityCloudNetHelper.getNextFallback(event.getPlayer(), event.getServer())
```

Comment:
As you called `handleConnectionFailed` there is no need of passing the server to the method, it simply generates an overload

Reply:
(Copy and Paste from my Comment to @juliarn's review) 
This actually fixed it for a single Lobby. This was my first try.

I guess I will revert this then. But why is it in the Bungeecord implementation when it shouldn't be here?